src/runtime/web/service.js

```diff
@@ -1,39 +1,45 @@
 'use strict';
```

Comment:
`runtime/web/index` still plugs the service in and starts the listeners. The instance itself is inert except for the language negotiation part.  I moved it to `runtime/` because I think that new runtimes will need their own services.  For instance, I imagine `runtime/nga` which could have a bridge.js-enabled service.js and a corresponding client.js.  You'd then pass the client instance to the view. Some runtimes won't need them at all (e.g. gaiabuild).

There's still some cleaning up to do:  service.js now requires stuff from bindings/html like langnego functions.  I think we can clean this up incrementally as we go.